src/layout.ts

```diff
@@ -113,13 +114,25 @@ export class BlocklyLayout extends PanelLayout {
   /**
    * Handle `after-attach` messages sent to the widget.
    */
-  protected onAfterAttach(msg: Message): void {
+  protected onAfterAttach(msg: Message): void {    
+    //inject Blockly with appropiate JupyterLab theme.
+    var jupyterlab_theme = define_jupyter_theme();
     this._workspace = Blockly.inject(this._host, {
-      toolbox: this._manager.toolbox
+      toolbox: this._manager.toolbox,
+      theme : jupyterlab_theme
```

Comment:
We can just set the Theme.

```suggestion
      theme: THEME
```